generator/transit_generator.cpp

```diff
@@ -44,6 +48,22 @@ using namespace std;
 
 namespace
 {
+bool CompStopsById(Stop const & s1, Stop const & s2)
+{
+  return s1.GetId() < s2.GetId();
+}
+
+/// \returns ref to a stop at |stops| by |stopId|.
+Stop const & FindStopById(vector<Stop> const & stops, StopId stopId)
+{
+  auto s1Id = equal_range(stops.cbegin(), stops.cend(),
```

Comment:
ASSERT is_sorted sops?

Reply:
I thought about CHECK. This method is called relevantly often ... ASSERT - ok. Let's put it here.